client/js/components/statement/splitStatement/SideBar.vue

```diff
@@ -70,7 +70,7 @@
 
         <div
           v-if="tagTopics.length"
-          class="flex-1 overflow-y-scroll mt-2 pr-1">
+          class="flex-1 overflow-y-scroll min-h-11 mt-2 pr-1">
```

Comment:
why do you set such a huge min-height here?

Reply:
because of the https://demoseurope.youtrack.cloud/issue/DPLAN-12253
it is noticeable in a few topics!
![Screenshot_2024-08-15_10-52-20](https://github.com/user-attachments/assets/a7aa68a9-428a-4e56-bb2a-14882fc5fe0c)

Reply:
I have pushed something here https://github.com/demos-europe/demosplan-core/pull/3562/commits/acc895cf92bf7085924056fec0075a34914c084b!